code/game/objects/items/devices/radio/intercom.dm

```diff
@@ -20,6 +20,7 @@
 	if(!current_area)
 		return
 	RegisterSignal(current_area, COMSIG_AREA_POWER_CHANGE, .proc/AreaPowerCheck)
+	update_icon()
```

Comment:
This is not necessary

Reply:
its just incase the intercom starts with one or the other on so it always has the right lights on

Reply:
It'll do it anyways even without this proc here, try it yourself if you don't believe me